torch/_torch_docs.py

```diff
@@ -3234,6 +3234,53 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, range=None, density=False) -> (Tensor, Tensor)
+histogram(input, bins, weights=None, density=False) -> (Tensor, Tensor)
+
+Computes the histogram of a tensor.
+
+Args:
+    {input}
+    bins (int or 1D Tensor): If int, defines the number of bins. If Tensor,
+        defines a monotonically increasing sequence of bin edges, allowing
+        for non-uniform bins. Default: 10.
+    range (tuple of floats, optional): Defines the lower and upper ends of the
+        range if bins is an integer.
+    weights (Tensor, optional): Weight for each value in the input tensor.
+        Should be of same size as input tensor.
+    density (bool): If ``False``, the result will contain the (weighted) number
+        of samples in each bin. If ``True``, the result is normalized to
```

Comment:
What would you think of adding more detail with an example like this:

If ``True``, the (weighted) number of samples is divided by both the sum of all (weighted) samples and the width of the bin. For example, in an unweighted histogram with 10 elements, if a bin contains 5 elements and has at edges (1, 3), then if `density=True` it is normalized from 5 to `5/(10 * (3 - 1)) = .25`. Conceptually this defines a piecewise-constant function over each bin whose integral over the bin is the percent of (weighted) samples in the bin and whose integral over the entire range is one.